components/logins/src/ffi.rs

```diff
@@ -45,6 +45,7 @@ pub mod error_codes {
     pub const INVALID_LOGIN_DUPLICATE_LOGIN: i32 = 64 + 2;
     pub const INVALID_LOGIN_BOTH_TARGETS: i32 = 64 + 3;
     pub const INVALID_LOGIN_NO_TARGET: i32 = 64 + 4;
+    pub const INVALID_LOGIN_UNSPECIFIED: i32 = 64 + 5;
```

Comment:
I'm personally fine with a generic "this is breaking because of something" error for now.